pkg/core/helper_test.go

```diff
@@ -71,8 +73,7 @@ func newBlock(index uint32, txs ...*transaction.Transaction) *Block {
 		},
 		Transactions: txs,
 	}
-	_ = b.rebuildMerkleRoot()
-	b.createHash()
```

Comment:
hm. why we don't have to call `createHash()` now?

Reply:
I guess because `Hash()` already does that internally.